packages/core/src/app/customer/Customer.tsx

```diff
@@ -212,6 +212,7 @@ class Customer extends Component<CustomerProps & WithCheckoutCustomerProps & Ana
                     privacyPolicyUrl={privacyPolicyUrl}
                     requiresMarketingConsent={requiresMarketingConsent}
                     step={step}
+                    useFloatingLabel={useFloatingLabel}
```

Comment:
Can you please rename all `useFloatingLabel` to `isFloatingLabelEnabled`?
We recently did this change to match the return flag from API in this PR, https://github.com/bigcommerce/checkout-js/pull/1230.